src/platforms/common/performance/index.mdx

```diff
@@ -79,18 +78,28 @@ Leaving the sample rate at `1.0` means that included instrumentation will send a
 </PlatformSection>
 
 <!-- notSupported here should be the opposite of whats in common/connect-services.mdx -->
+
 <PlatformSection notSupported={["javascript", "javascript.cordova"]}>
 
 ## Connecting Services
 
 If you are also using Performance Monitoring for [JavaScript](/platforms/javascript/performance/), depending on where your request originates, you can connect traces:
+
 1. For requests that start in your backend, by [adding a meta tag](/platforms/javascript/performance/connect-services/#pageload) in your HTML template that contains tracing information.
 2. For requests that start in JavaScript, by the SDK [setting a header](/platforms/javascript/performance/connect-services/#navigation-and-other-xhr-requests) on requests to your backend.
 
 Otherwise, backend services with Performance Monitoring connect automatically.
 
 </PlatformSection>
 
+<PlatformSection supported={["javascript",]} notSupported={["react-native"]}>
```

Comment:
```suggestion
<PlatformSection supported={["javascript.react"]}>
```

We should scope this to react only. (That should also fix all of those broken links above.)